### ISSUE TITLE
Display mode icons

### ISSUE THREAD

**Issue by thibaudk** [ossia/score#1056]
![BLicons](https://user-images.githubusercontent.com/26526517/92133912-37698500-ee09-11ea-8531-1fe19079ce12.png)

This has been discussed else where but All of them should be addressed at once I think for consistency.

* linear/nodal icon is backwards. IMO it should show the nodal option when in linear mode and inversely

* "Musical" and "show cable" do not have an alternative icon, I think it would make more sense than changing color, as this is already the behavior when hovering.

**Comment by jcelerier** [comment]
@aklevy would you like to tackle that one ? or I can look into it if you don't have time

**Comment by aklevy** [comment]
@thibaudk  @jcelerier  oops, sorry for the late reply, ok for the first point, but for the second point can you clarify a bit what you think should be done ?

**Comment by jcelerier** [comment]
for the second I imagine something like : 
- the semiquavers being strikethrough maybe ? 
- the cable disappearing in "no cables" mode ? 

But I think there should be a bit of discussion before rushing it

**Comment by aklevy** [comment]
hmm, ok we will see for the second one.  Meanwhile, the first issue is done.

**Comment by thibaudk** [comment]
>     * the semiquavers being strikethrough maybe ?
 Could it be something as simple as the "≈" symbol ?

>     * the cable disappearing in "no cables" mode ?
yes, like the same icon but with a doted line.

> hmm, ok we will see for the second one. Meanwhile, the first issue is done.
 :+1:

**Comment by jcelerier** [comment]
resolution: disable hovering when the button is selected, onyl enable it when it is not

**Comment by HugoGenillier** [comment]
![modes-ezgif com-video-to-gif-converter](https://github.com/ossia/score/assets/118723648/d81dea7a-bd67-4d35-a081-924337ab5fe2)
In fact having the same behavior for each mode switch will be more consistent. 
Is the idea of having alternative icons for wireless mode and musical mode still on track ? With that and by deleting the color change on click, they will behave like the temporal/nodal mode icon which will improve the coherence for users.

Also, the inspector displays the same message for each icons when hovered :

- temporal/nodal : "Change between nodal and timeline mode"
- default/musical : "Enable musical mode"
- cable/cableless : "Show cables (Alt+Shift+G)"

Maybe having a different inspector's description for each icon will help the understanding of each mode for users.
Like one text for temporal, one for nodal, one for musical etc...  

Perhaps I can find the right alternative icons, the doted line seems like a great start.

**Comment by jcelerier** [comment]
whaddya think @thibaudk @aklevy ?

**Comment by thibaudk** [comment]
Yes, I still think all 3 should toggle between 2 different icons.
